lib/taste_tester/commands.rb

```diff
@@ -99,7 +101,7 @@ def self.test
           tested_hosts << hostname
         end
       end
-      unless TasteTester::Config.skip_post_test_hook
+      unless TasteTester::Config.skip_post_test_hook || TasteTester::Config.linkonly
```

Comment:
I've contemplated this but I think I want to leave it in.  My reasoning: our default hook functions want a repo, but I think the "spirit" of linkonly doesn't involve a repo (i.e. the help is "Only setup the remote server, skip uploading.")